From 07f3da2e9d3315860b9a92d4acd6530af634662f Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Sat, 11 Jan 2020 17:08:26 -0800
Subject: [PATCH] Fix compatilibity with gpsd-3.19

* fixes:
gpsd-client/1.0.0-1-r0/git/src/client.cpp:183:22: error: 'struct gps_data_t' has no member named 'epe'
  183 |         fix.err = p->epe;
      |                      ^~~
ninja: build stopped: subcommand failed.

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 src/client.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/client.cpp b/src/client.cpp
index 32fcb44..f449be2 100644
--- a/src/client.cpp
+++ b/src/client.cpp
@@ -175,7 +175,11 @@ class GPSDClient {
         fix.gdop = p->gdop;
 #endif
 
+#if GPSD_API_MAJOR_VERSION >= 8
+        fix.err = p->fix.sep;
+#else
         fix.err = p->epe;
+#endif
         fix.err_vert = p->fix.epv;
         fix.err_track = p->fix.epd;
         fix.err_speed = p->fix.eps;
-- 
2.17.1

